internal/stack.bzl

```
@@ -0,0 +1,101 @@
+# Copyright 2020 The Bazel Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""PostCSS stack rule.
+
+Using postcss_stack allows reuse of PostCSS plugin configuration. (Without
+resorting to methods such as top-level constants.)
+
+When postcss_stack is used in conjunction with workers for postcss_binary,
+the worker process will be able to cache instantiated PostCSS plugins.
+"""
+
+load("@bazel_skylib//lib:dicts.bzl", "dicts")
+load(":runner_bin.bzl", "postcss_runner_bin")
+
+PostcssPluginsInfo = provider("""Provides a list of plugins.""", fields = ["plugins"])
```

```suggestion
PostcssPluginsInfo = provider("Provides a list of plugins.", fields = ["plugins"])
```

You don't need the triple quotes if the string isn't multiline.

---

Fixed, thanks!